### ISSUE TITLE
Error on Deno Deploy using serveFileWithTs

### ISSUE THREAD

**Issue by BrunoBernardino** [ayame113/ts-serve#9]
First of all, thank you so much for this great library! I've implemented it in my [Simple Deno Website Boilerplate](https://github.com/BrunoBernardino/deno-boilerplate-simple-website), but it's failing on Deno Deploy:

```
Uncaught SyntaxError: The requested module 'https://deno.land/std@0.144.0/http/file_server.ts' does not provide an export named 'ServeDirOptions'
    at https://deno.land/x/ts_serve@v1.1.0/mod.ts:1:20
```

![Screenshot from 2022-07-30 10-51-30](https://user-images.githubusercontent.com/1239616/181905349-154b8316-200e-48d7-9159-17e8b76aacb3.png)

It's quite weird because I can see the export does exist in `std` and has for a while, maybe it's a Deno Deploy runtime issue ([I've asked in the Discord server](https://discord.com/channels/684898665143206084/826085979344470037/1002878942362996856) and also created an issue at https://github.com/denoland/deploy_feedback/issues/252), but wanted to log here in case someone else sees it.

**Comment by BrunoBernardino** [comment]
I was able to fix it for myself by [implementing a simpler serving of the files](https://github.com/BrunoBernardino/deno-boilerplate-simple-website/commit/305c1ff033cca9e5a8308210d4e31afe87f8cbc3) but this will likely affect others trying to use this library.

**Comment by ayame113** [comment]
Thanks for letting me know about this issue!

I have seen this error before and remember fixing it. Perhaps if I release a new version everything should be fine. wait a moment please.

**Comment by ayame113** [comment]
This issue has been fixed in https://github.com/ayame113/ts-serve/pull/7/files#diff-e8c3622ccfbed67f4b3058cd4417bf41d066236f49c6f3e63b7c243b584c6734R1-R6 and I just released a new version (v1.2.0) with that fix so I think this can be closed.
